src/main/java/com/google/devtools/build/lib/rules/apple/DottedVersion.java

```diff
@@ -129,8 +129,11 @@ public static Option option(DottedVersion version) {
   private static final Splitter DOT_SPLITTER = Splitter.on('.');
   private static final Pattern COMPONENT_PATTERN =
       Pattern.compile("(\\d+)([a-z0-9]*?)?(\\d+)?", Pattern.CASE_INSENSITIVE);
+  private static final Pattern COMMENTIVE_COMPONENT_PATTERN =
```

Comment:
Can we change "commentive" to "descriptive"?

Reply:
Done